myapp/static/myapp/css/base.css

```
@@ -1,53 +1,81 @@
-html{
+html {
     font-size: 16px;
     font-family: Arial, Helvetica, sans-serif;
 }
-body{
+
+body {
     margin: 0;
+    position: relative;
 }
 
 /* ヘッダー */
-.header{
-    width: 100vw;
+.header {
+    width: 100%;
     height: 64px;
     border-bottom: solid 1px #9b9b9b;
+    background-color: #ececec;
 }
-.header-container{
+
+.header-container {
     position: relative;
     width: 90%;
     height: 100%;
     margin-left: auto;
     margin-right: auto;
 }
-.header-container__title{
+
+.header-container__title {
     position: absolute;
-    top: 0;
+    top: 30%;
     bottom: 0;
     left: 0;
     right: 0;
     height: 1rem;
     line-height: 1rem;
-    margin: auto;
+    margin-top: 7px;
     text-align: center;
     font-size: 1.2rem;
     color: #000;
 }
 
+.container {
+    width: 400px;
+    height: 100vh;
+    border: 1px solid black;
+    justify-self: center;
+    overflow-y: scroll;

```

これでははみ出ていない場合にも常にスクロールバーが表示されてしまうので以下のようにすることが一般的です。
```suggestion
    overflow-y: auto;
```